app/hallebarde/responses/authorizer_responses.py

```diff
@@ -0,0 +1,16 @@
+def generate_policy(is_valid: bool, resource: str) -> dict:
```

Comment:
I would call it `generate_aws_policy` to enforce the fact that this format comes from AWS and it is mandatory to respect it